src/shared/options.ts

```diff
@@ -398,6 +400,12 @@ class LanguageServerOptionsImpl implements LanguageServerOptions {
     public get compilerDiagnosticScope() {
         return readOption<string>('dotnet.backgroundAnalysis.compilerDiagnosticsScope', 'openFiles');
     }
+    public get componentPaths() {
+        return readOption<{ [key: string]: string }>('dotnet.server.componentPaths', {});
+    }
+    public get enableXamlToolsPreview() {
+        return readOption<boolean>('dotnet.enableXamlToolsPreview', false);
```

Comment:
Isn't this supposed to be true now?

Reply:
ah, its true in the package.json, but I'll update it here too.